packages/website/src/pages/index.tsx

```diff
@@ -107,9 +108,13 @@ const features: FeatureItem[] = [
 
 function Feature({ title, description }: FeatureItem): JSX.Element {
   return (
-    <div className="col col--12 padding-vert--lg">
-      <h2 className="text--center">{title}</h2>
-      {description}
+    <div className="col col--12 padding-vert--lg ">
+      <div className="text--center">
+        <Heading as="h2" id="title">
+          {title}
+        </Heading>
+      </div>
```

Comment:
Heh, I filed https://github.com/facebook/docusaurus/issues/8327 that we should be able to put the `className="text--center"` on the `Heading` (but can't right now).

Reply:
So I moved the class name to div, should I modify it?

Reply:
Oh sorry I didn't mean to imply you need to do anything. This is perfect. Docusaurus just needs to fix a bug (or tell me I'm wrong about it being a bug 😂).